stable/yugabyte/templates/service.yaml

```diff
@@ -218,10 +218,19 @@ spec:
           postStart:
             exec:
               command:
-                - "sh"
+                - "bash"
                 - "-c"
                 - >
                   mkdir -p /mnt/disk0/cores;
+                  pattern="$(sysctl kernel.core_pattern --values || true)";
```

Comment:
if the sysctl fails, pattern is an empty string right, is that fine?

Reply:
Yes, that's fine. It won't go to the inside block where mkdir and ln is run.